### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by DanaBrasanac1901**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,26 @@
 .App {
```
Comment (SaleBojke) [inline src/App.css]: put this one to lower case, use kebab-case for naming of css classes, like you did in the rest of the file.

File: src/App.tsx [hunk 2]
```diff
@@ -1,25 +1,41 @@
-import logo from './logo.svg';
-import './App.css';
+import './App.css'
```
Comment (SaleBojke) [inline src/App.tsx]: move css import to the bottom of imports.

File: src/App.tsx [hunk 3]
```diff
@@ -1,25 +1,41 @@
...
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+    <div className='App'>
+      <header className='App-header'>
```
Comment (SaleBojke) [inline src/App.tsx]: Change App-header class to kebab-case as well.

File: src/App.tsx [hunk 4]
```diff
@@ -1,25 +1,41 @@
...
+        <ConditionalWrapper
```
Comment (SaleBojke) [inline src/App.tsx:18]: 👌

File: package.json [hunk 5]
```diff
@@ -20,7 +22,10 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src/**/*.{ts,tsx}",
+    "lint:fix": "eslint --fix src/**/*.{js,jsx,ts,tsx,json}",
```
Comment (aleksandarkrnjaic) [inline package.json]: you probably don't need js files. You decide whether you want to keep it.

File: src/styles/SearchBar.css [hunk 6]
```diff
@@ -0,0 +1,15 @@
+.search_bar_input{
+    font-size: large;
+    height: 27px;
+    margin-right: 5px;
+    border-radius: 15px;
+}
+
+.search_button{
```
Comment (aleksandarkrnjaic) [inline src/styles/SearchBar.css]: If you want to follow BEM more closely use two underscores instead of one

**Review by aleksandarkrnjaic — APPROVED** [review]

**Pull request merged**